.htmltest.yml

```diff
@@ -8,6 +8,8 @@ CheckMailto: false
 TestFilesConcurrently: true
 IgnoreDirs:
   - ^blog/(\d+/)?page/\d+
+  # TODO drop after https://github.com/open-telemetry/opentelemetry.io/issues/5267 is fixed:
+  - zh/docs/contributing/pr-checks
```

Comment:
@open-telemetry/docs-zh-approvers - this temporary ignore rule will need to be removed as a part of the fix to

- #5267